dll/win32/shell32/dialogs/filetypes.cpp

```diff
@@ -28,6 +28,8 @@ WINE_DEFAULT_DEBUG_CHANNEL (fprop);
 
 /////////////////////////////////////////////////////////////////////////////
 
+EXTERN_C BOOL PathIsExeW(LPCWSTR lpszPath);
```

Comment:
Which filetypes are being considered as "executable exensions"? exe,com,cmd,bat,lnk,pif.... anything else? I do recommend to add the list of intended exclusions to the PRs description-field.

Reply:
https://github.com/reactos/reactos/blob/master/dll/win32/shell32/wine/shellpath.c#L539

Reply:
Thank you Giannis. That is at the time of me writing this reply {L"exe", L"com", L"pif", L"cmd", L"bat", L"scf", L"scr", L"" }; and I do confirm that all of those are hidden on my XPSP3. So the proposed change could be correct. So please add that to the description field and you can resolve this conversation.

Reply:
So .lnk is not hidden yet, although it is hidden in my XPSP3.

Reply:
> So .lnk is not hidden yet

Are you sure? It's hidden for me, ROS sets EditFlags=FTA_Exclude.

Reply:
You are correct. Lnk is hidden. But it is hidden, because it was hidden already before your PR. Not due to your code changes, and my last comment was based solely on what I could see in the proposed code-changes. For those who are interested in what ros currently shows without PR7452 applied yet: [before_PR7452.webm](https://github.com/user-attachments/assets/235fa69d-de20-4fa5-b400-c091c65631f9)